packages/salesforcedx-utils-vscode/test/jest/helpers/paths.test.ts

```diff
@@ -70,4 +73,43 @@ describe('test project paths', () => {
       expect(relativeStateFolder).toEqual(FAKE_STATE_FOLDER);
     });
   });
+
+  describe('test SFDXGlobalConfigPath', () => {
+    it('should return the global config path', () => {
+      const mockOS = jest.requireActual('os');
+      mockOS.homedir = jest.fn().mockReturnValue(FAKE_HOME_DIR);
```

Comment:
in a huddle we figured out that this was altering the os.homedir function for all tests.  An FYI if we have to use this method in the future